Main.c

```diff
@@ -0,0 +1,72 @@
+//Attachments area
```

Comment:
Please check c coding standard. There are rules on indentation, variable naming and Commenting of the code.

Don't put commenting for later as it will be tedious task. Comment the code as you go